### PR TITLE
fix-BVCZ

### LINKED ISSUE #287: BVCZ
| Key | Value |
| --- | --- |
| ID | BVCZ |
| Source | Vaaran Bhai Gurdas Ji |
| Page | 15 |
| Line | ```qwry hoie Agws ciV; EVik quiV quiV Qwn hlcy [``` |
| Correction | ```quiV quiV``` ≠ ```quit quit```  |

PROOF (EDITION)

Image:
vaaran73

### PR DESCRIPTION
**Pull request by bhajneet**
Closes #287

### CODE REVIEW

File: data/Vaaran Bhai Gurdas Ji/15.json [hunk 1]
```diff
@@ -2179,7 +2179,7 @@
         "id": "C3K5",
         "source_page": 15,
         "source_line": 12,
-        "gurmukhi": "pwirjwqu guru Cif kY; mMgin klpqroˆ Pl kcy [",
+        "gurmukhi": "pwirjwqu guru Cif kY; mMgin klp qroˆ Pl kcy [",
```
Comment (Sarabveer) [inline data/Vaaran Bhai Gurdas Ji/15.json]: Need reference

File: data/Vaaran Bhai Gurdas Ji/15.json [hunk 2]
```diff
@@ -2248,7 +2248,7 @@
         "id": "BVCZ",
         "source_page": 15,
         "source_line": 12,
-        "gurmukhi": "qwry hoie Agws ciV; EVik quiV quiV Qwn hlcy [",
+        "gurmukhi": "qwry hoie Agws ciV; EVik quit quit Qwn hlcy [",
```
Comment (Sarabveer) [inline data/Vaaran Bhai Gurdas Ji/15.json]: Approved ✔️

File: data/Vaaran Bhai Gurdas Ji/15.json (hunk 1 above)
Comment (Sarabveer) [reply data/Vaaran Bhai Gurdas Ji/15.json]: Should be `qroN`

Comment (bhajneet) [reply data/Vaaran Bhai Gurdas Ji/15.json]: This line should not have been changed in this commit/PR. Fixed with another commit.

**Pull request merged**